macOS/DuckDuckGo/NavigationBar/View/AddressBarButtonsViewController.swift

```diff
@@ -397,21 +395,16 @@ final class AddressBarButtonsViewController: NSViewController {
 
         if featureFlagger.isFeatureOn(.aiChatSidebar),
            aiChatMenuConfig.openAIChatInSidebar,
+           !isTextFieldEditorFirstResponder,
```

Comment:
Added due to simplification of "Ask Duck.ai" no longer opening sidebar with prompt but always a full tab Duck.ai, so toggle the sidebar only when address bar is not in focus.